### DISCUSSION TITLE
Bridged connection stopped working after I updated to macOS 15.0.1

### DISCUSSION THREAD

**Discussion by haulesbaules** [utmapp/UTM#6717] [macOS] [answered]
I run a macOS emulation on a MacBook Pro M1. Since I updated the host to macOS 15.0.1 the bridged network doesn't work anymore. 
Is this a coincidence? 
Shared network works like usual but I need to run VPNs on both machines so it's no good for my use case.
This happened to anyone else? Any fix for it?

**Comment by haulesbaules** [comment, marked as answer]
Seems like after uninstalling UTM couple of times it's working again.

**Comment by 089apps** [comment]
bridged network still not working on my Macbook Air, M3, Sonoma 14.6
